.github/workflows/test.yml

```diff
@@ -53,6 +53,8 @@ jobs:
         swiftver:
           - swift:5.2
           - swift:5.5
+          - swift:5.6
+          - swiftlang/swift:nightly-main
```

Comment:
I don't think this will work, there's a regression with the compiler and Vapor

Reply:
Apparently Fluent by itself doesn't hit whatever's borked 🤷‍♀️